lib/activity_runtime_api.rb

```diff
@@ -157,7 +157,8 @@ def self.publish_sequence(hash, user)
         :publication_status => "published",
         :user => user,
         :author_email => hash["author_email"],
-        :is_locked => hash["is_locked"]
+        :is_locked => hash["is_locked"],
+        :material_type => Investigation.name
```

Comment:
> There is also code in this file for handling the publishing of Activities, so that should set the material_type to 'Activity'.

This is already the default value right?

Reply:
With the new changes to set the material_type automatically, I think we can drop this manual setting of material_type.